wordpress-mobile/WordPress-iOS
# Update Podfile to support RN-0.61

This PR updates the RN libraries references in the podfile Gutenberg section to make sure they follow the new names and needs of the new version.

To test:
 - Delete the pods folder: `rm -Rf Pods`. This updates brings a a lot of pod change for RN so it's better to start with a clean folder.
 - Run `rake dependencies`
 - Make sure the project builds
 - Make sure you can open a GB post.
 - Smoke test the functionality on the GB editor.

PR submission checklist:
